docs/tables/aws_config_rule.md

```diff
@@ -44,3 +44,14 @@ from
 where
   name Like '%s3-bucket%';
 ```
+
+### List complaince types by config rule
+
+```sql
+select
+  name as config_rule_name,
+  compliance_status -> 'Compliance' -> 'ComplianceType' as compliance_type
+from
+  aws_config_rule,
+  jsonb_array_elements(compliance_by_config_rule) as compliance_status;
+```
```

Comment:
Add another example above this to depict the date that comes in the  `compliance_by_config_rule` for clarity
```
select jsonb_pretty(compliance_by_config_rule) from aws_config_rule where name = 'approved-amis-by-id'
```